doc-content/enterprise-only/openshift/githooks-proc.adoc

```diff
@@ -0,0 +1,80 @@
+[id='githooks-proc']
+= Providing the Git hooks directory
+
+If you configure the `GIT_HOOKS_DIR` parameter, you must provide a directory of Git hooks. You must mount this directory on the {CENTRAL} deployment. 
+
+The typical use of Git hooks is interaction with an upstream repository. To enable Git hooks to push commits into an upstream repository, you must also provide a secret key that corresponds to a public key configured on the upstream repository.
+
+.Procedure
+
+. If pushing commits to an upstream repository is required, complete the following steps to enable access to the repository:
+.. Create a pair of a secret key named `id_rsa` and public key named `id_rsa.pub`. Use an email address that can access the upstream repository. Set an empty passphrase. For instructions, see https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/#generating-a-new-ssh-key[Generating a new SSH key]. 
+.. Upload the public key to the upstream repository.
+.. Create an OpenShift secret named `{PRODUCT_INIT}-centr-gitkey` from the secret key:
++
+[subs="attributes,verbatim,macros"]
+----
+oc create secret generic --from-file=id_rsa=id_rsa {PRODUCT_INIT}-centr-gitkey
+----
++
+.. Mount the secret in the ssh key path of the {CENTRAL} deployment:
++
+[subs="attributes,verbatim,macros"]
+----
+oc set volume dc/<myapp>-{PRODUCT_INIT}centr --add --type secret --secret-name {PRODUCT_INIT}-centr-gitkey --mount-path=/home/jboss/.ssh --name=ssh-key
+----
++
+Where `<myapp>` is the application name that was set when configuring the template. 
```

Comment:
Well, AFAIK, we really aren't supposed to use sentence frags like this. I don't like it either, but...
In the preceding example, myapp... ?